### DISCUSSION TITLE
Does it work with HmIP-MOD-HO?

### DISCUSSION THREAD

**Discussion by reinisb** [thkl/hap-homematic#277] [General]
Hi folks! I am planning to buy the [HmIP-MOD-HO](https://www.homematic-ip.com/en/products/detail/homematic-ip-module-for-hoermann-drives.html) adapter and connect it to my Hörmann Rotamatic swing gate. Then, I intend to use hap-homematic in my RaspberryMatic to control and see the status of the swing gate via the Home app on my iPhone.

HmIP-MOD-HO is a universal adapter that works with Hörmann garage doors as well as with Hörmann swing gates (like the Rotamatic I have). In my case, it would be a box that plugs directly into the bus port of the Rotamatic swing gate control unit and allows to open/close the swing gate as well as to see whether the gate is open (there is no separate sensor device, it is a single little box).

My question is - does hap-homematic support HmIP-MOD-HO?
Also, as far as I can tell, Apple's HAP does not have an accessory type for swing gates, but I suppose as a workaround swing gates usually use the garage door type - would it also apply in this case?

**Comment by gerd2005** [comment]
Hi, yes it works fine!

![EB6C7AE7-C654-4512-8746-A6077A66E80F](https://user-images.githubusercontent.com/72799068/101778846-b8b17180-3af4-11eb-8dee-b3f29757a91b.png)

**Comment by reinisb** [comment]
@gerd2005 - thanks for the quick response - looks great! Even shows the gate status. Did it require any custom configuration or did it work out of the box?

**Comment by gerd2005** [comment]
But notice, that not every Channel is completely supported yet.
![537D8A62-3D15-416D-ABBD-FA8530499D2C](https://user-images.githubusercontent.com/72799068/101821473-48bcde80-3b28-11eb-8aae-87c0038544ce.png)

**Comment by gerd2005** [comment]
And maybe you need the HCP-Adapter. Ist depends on which Hörmann Product you use.

**Comment by reinisb** [comment]
@gerd2005 Have you experienced an issue like [this](https://github.com/thkl/hap-homematic/issues/294)?
